.editorconfig

```diff
@@ -1,2 +1,3 @@
 [*.{kt,kts}]
```

Comment:
> The app is not allowed to open the intent url dialog without the app being currently open anymore. I don't know if there is a solution for that, but I haven't found one so far.

Have you enabled the "Intent Action Permission" in the settings? Or does this setting now have no effect anymore?

Reply:
Hmm, testing this again and it seems to work now.  
Maybe I really just didn't toggle it last time.